src/main/kotlin/net/atos/zac/app/signalering/SignaleringRestService.kt

```diff
@@ -80,29 +74,63 @@ class SignaleringRestService @Inject constructor(
         }
     }
 
+    /**
+     * Lists zaken signaleringen for the given signaleringsType.
+     */
+    @GET
+    @Path("/zaken/{type}")
+    fun listZakenSignaleringen(
+        @PathParam("type") signaleringsType: SignaleringType.Type,
+        @QueryParam("pageNumber") @DefaultValue(value = "0") pageNumber: Int,
+        @QueryParam("pageSize") @DefaultValue(value = "5") pageSize: Int
+    ): Response =
+        signaleringService.countZakenSignaleringen(signaleringsType).let { objectsCount ->
+            if (pageNumber > objectsCount.maxPages(pageSize)) {
+                return Response.status(Response.Status.NOT_FOUND).build()
+            }
+            Response.ok()
+                .header(TOTAL_COUNT_HEADER, objectsCount)
+                .entity(signaleringService.listZakenSignaleringenPage(signaleringsType, pageNumber, pageSize))
+                .build()
+        }
+
     @GET
     @Path("/taken/{type}")
+    @Deprecated("default page size is to be changed when we switch to paging")
```

Comment:
This is a comment on future changes, not a method deprecation, right? then I would probably remove it completely.

Reply:
agreed